packages/camera/camera_android_camerax/lib/src/camera_info.dart

```diff
@@ -42,14 +51,31 @@ class CameraInfoHostApiImpl extends CameraInfoHostApi {
   /// Maintains instances stored to communicate with native language objects.
   late final InstanceManager instanceManager;
 
-  /// Gets sensor orientation degrees of [CameraInfo].
+  /// Gets sensor orientation degrees of the specified [CameraInfo] instance.
```

Comment:
Consider linking to the camerax documentation of sensor orientation.

Reply:
In the actual class, the documentation is linked, so I'm going to leave as is to avoid redundancy.

Reply:
Is this the object that dart callers will interact with?

Reply:
No, they'll interact with the `CameraInfo` object.

Reply:
This class should probably be private, honestly

Reply:
> No, they'll interact with the `CameraInfo` object.

This is a little confusing since this is camera_info.dart but consider my comment resolved. 

Nonblocking for this pr. You might want to consider a pass where you make objects like this private or reduce their scope. After this is published those will be breaking changes.